modules/service-account-auto-secret-removed.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * authentication/using-service-accounts-in-applications.adoc
+// * pods/nodes-pods-secrets.adoc
+
+:_content-type: CONCEPT
+[id="auto-generated-sa-token-secrets_{context}"]
+= About automatically generated service account token secrets
+
+When a service account is created, a service account token secret is automatically generated for it. This service account token secret, along with an automatically generated Docker configuration secret, is used to authenticate to the internal {product-title} registry. Do not rely on these automatically generated secrets for your own use; they might be removed in a future {product-title} release.
```

Comment:
Should the `Do not rely on these automatically generated secrets for your own use; they might be removed in a future {product-title} release.` portion of this be in a warning or call out ?

Reply:
It could potentially be, but we usually try to avoid having multiple notes/warnings back-to-back. Let me know if you feel strongly that it would be better received as a note, and I can make an exception to that rule.